### DISCUSSION TITLE
ccf-validation-tools/Small_intestine/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#209] [General]
# ccf-validation-tools/Small_intestine/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Small_intestine/comments/

**Comment by emquardokus** [comment]
**Fixed in Editable Small_Intestine_v1.2_DRAFT:**  https://docs.google.com/spreadsheets/d/1KMOjJj-bVAqo39KuIOo0SaIVRtW9-5C5Y65NwnXlGqw/edit#gid=247140941
**Different labels:** 
In row 65, the term CL:0000186 to myofibroblast cell
In row 63, the term CL:0000115 to endothelial cell
**Blank ontology ID:**
These will nee more scrutiny for location specific terms which have now been added to uberon since table was originally made.
**Terms from another ontology**
FMA:7206 only has 4 numbers after the  colon, but is a valid FMA ID despite validation report stating "N is a number and it should have exact 5 number after the colon."
Rows 55-96 FMA:7206 duodenum was replaced with superior part of duodenum	FMA:14926; Uberon does not provide sub-regions of duodenum.

**Comment by emquardokus** [comment]
**Blank ontology ID**
Rows 13-39 added specific Mucosa of duodenal ampulla	FMA:17101;  this term is not present in uberon
Rows 40-44 added specific Submucosa of duodenal ampulla	FMA:17107; this term is not present in uberon
Rows 45-51 added specific Muscle layer of duodenal ampulla	FMA:17113; Muscle layer of duodenal ampulla	FMA:17113
Rows 213-219 added specific muscularis propria Muscle layer of ascending part of duodenum	FMA:17116

**Terms from another ontology:**
Rows 97-138 uberon does not contain all of the different sections of the duodenum, whereas FMA ontology does, which is why this was chosen: descending part of duodenum FMA:14928.
Rows 139-180 uberon does not contain all of the different sections of the duodenum, whereas FMA ontology does, which is why this was chosen:horizontal part of duodenum	FMA:14929
Rows 181-222 uberon does not contain all of the different sections of the duodenum, whereas FMA ontology does, which is why this was chosen: ascending part of duodenum	FMA:14930

**Comment by anitacaron** [comment]
New terms in the ontology:
Small intestine :  jejunum lamina propria (UBERON:8600034)
Small intestine : ileum lamina propria (UBERON:8600035)
